pkg/webui/components/button/index.js

```diff
@@ -178,16 +164,30 @@ const Button = forwardRef((props, ref) => {
 
   const htmlProps = { autoFocus, name, type, value, title, onBlur, form, ...dataProps }
   const buttonClassNames = assembleClassnames(props)
-  return (
+
+  const buttonElement = (
     <button
       className={buttonClassNames}
       onClick={handleClick}
       children={buttonChildren({ ...props, expanded })}
       disabled={busy || disabled}
-      ref={ref}
+      ref={combinedRef}
       {...htmlProps}
     />
   )
+
+  if (dropdownItems) {
+    return (
+      <div className="pos-relative">
+        {buttonElement}
+        <Dropdown className={classnames(dropdownClassName)} open={expanded}>
+          {dropdownItems}
+        </Dropdown>
+      </div>
+    )
+  }
```

Comment:
Note that the dropdown is now rendered outside of the button. The issue with rendering the dropdown within the button is that the entire dropdown will be semantically considered as a button, which results in issues such as having the pointer cursor rendered regardless of whether an item in the dropdown is clickable. Likewise, it will always trigger the button onClick event when an item in the dropdown is clicked.

Reply:
cc @PavelJankoski